CRM/Osdi/Form/OSDIImport.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use CRM_Osdi_ExtensionUtil as E;
+
+/**
+ * Form controller class
+ *
+ * @see https://wiki.civicrm.org/confluence/display/CRMDOC/QuickForm+Reference
+ */
+class CRM_Osdi_Form_OSDIImport extends CRM_Core_Form {
```

Comment:
QuickForm sucks and everyone hates it.  Consider whether you want to extend the Page class instead and use AJAX-y widgets to update your various settings.  Let's discuss this before you get too far into QuickForm.